scripts/ci/trigger_docs_update

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#
+# Trigger the job that will update the documentation website.
+# Argument:
+#   version: the version of the new release.
+
+set -ev
+
+if [ "$#" -lt 1 ]; then
+    echo "Usage: $0 <version>"
+    exit 1
+fi
+
+readonly VERSION="$1"
+
+readonly BODY="{
+    \"request\": {
+        \"branch\": \"master\",
+        \"config\": {
+            \"env\": [\"VERSION="${VERSION}"\"]
+        }
+    }
+}"
+
+curl -s -X POST \
+   -H "Content-Type: application/json" \
+   -H "Accept: application/json" \
+   -H "Travis-API-Version: 3" \
+   -H "Authorization: token "${DOCS_TRAVIS_API_TOKEN}"" \
```

Comment:
There should be no quotes around `${DOCS_TRAVIS_API_TOKEN}`.

Reply:
ok